vcd/resource_vcd_vapp_vm_test.go

```diff
@@ -54,6 +55,8 @@ func TestAccVcdVAppVm_Basic(t *testing.T) {
 					testAccCheckVcdVAppVmExists(vappName2, vmName, "vcd_vapp_vm."+vmName, &vapp, &vm),
 					resource.TestCheckResourceAttr(
 						"vcd_vapp_vm."+vmName, "name", vmName),
+					resource.TestCheckResourceAttr(
```

Comment:
I think the read function should be updated just enough to deal with this field, or else the test is meaningless

Reply:
I would, but decided not, cause I would need new function in govcd to fetch VM guest customization and this changes are in your future PR.

Reply:
If this is urgent, I suggest doing it within this PR. I still have two projects to finish before working on vApp/VM.

Reply:
https://github.com/vmware/go-vcloud-director/pull/244